README.md

```diff
@@ -216,6 +216,10 @@ __Sample of a real Grabbit Job status__
 
 ![alt text](assets/ExampleGrabbitStatus.jpg)
 
+Two loggers are predefined for Grabbit. One for Grabbit Server and the other for Grabbit Client. 
```

Comment:
Maybe too much info?  I think the majority of people reading the README are users, rather than Grabbit developers.